docs/guides/videojs.md

```diff
@@ -63,7 +63,7 @@ var html5 = new Html5(options);
 var Html5 = videojs.getTech('Html5');
 var MyTech = videojs.extend(Html5, {});
 // Register the new Tech
-VjsButton.registerTech('Tech', MyTech);
+videojs.registerTech('MyTech', MyTech);
```

Comment:
`videojs.registerTech()` or `Html5.registerTech()` work here (but not `MyTech.registerTech()`. Which is prefered?